docs/helm/eks-deployment.md

```diff
@@ -0,0 +1,315 @@
+      verbs: ["get","watch","list"]
+    - apiGroups: [""]
+      resources: ["nodes"]
+      verbs: ["list","watch"]
+    ---
+    apiVersion: rbac.authorization.k8s.io/v1beta1
+    kind: ClusterRoleBinding
+    metadata:
+      name: external-dns-viewer
+    roleRef:
+      apiGroup: rbac.authorization.k8s.io
+      kind: ClusterRole
+      name: external-dns
+    subjects:
+    - kind: ServiceAccount
+      name: external-dns
+      namespace: kube-system
+    ---
+    apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      name: external-dns
+    spec:
+      strategy:
+        type: Recreate
+      selector:
+        matchLabels:
+          app: external-dns
+      template:
+        metadata:
+          labels:
+            app: external-dns
+        spec:
+          serviceAccountName: external-dns
+          containers:
+          - name: external-dns
+            image: registry.opensource.zalan.do/teapot/external-dns:latest
+            args:
+            - --source=service
+            - --domain-filter=YOUR-DOMAIN-NAME
+            - --provider=aws
+            - --policy=sync
+            - --aws-zone-type=public
+            - --registry=txt
+            - --txt-owner-id=acs-deployment
+            - --log-level=debug
+    ```
+
+4. Use the kubectl command to deploy the external-dns service.
+
+   ```bash
+   kubectl apply -f external-dns.yaml -n kube-system
+   ```
+
+5. Find the name of the role used by the nodes by running the following command (replacing `YOUR-CLUSTER-NAME` with the name you gave your cluster):
+
+    ```bash
+    aws eks describe-nodegroup --cluster-name YOUR-CLUSTER-NAME --nodegroup-name linux-nodes --query "nodegroup.nodeRole" --output text
+    ```
+
+6. In the [IAM console](https://console.aws.amazon.com/iam/home) find the role discovered in the previous step and attach the "AmazonRoute53FullAccess" managed policy as shown in the screenshot below:
+
+    ![Attach Policy](./diagrams/eks-attach-policy.png)
+
+### File System
+
+1. Create an Elastic File System in the VPC created by EKS using [these steps](https://docs.aws.amazon.com/efs/latest/ug/creating-using-create-fs.html) ensuring a mount target is created in each subnet. Make a note of the File System ID (circled in the screenshot below).
+
+    ![EFS](./diagrams/eks-efs.png)
+
+2. Find The ID of VPC created when your cluster was built using the command below (replacing `YOUR-CLUSTER-NAME` with the name you gave your cluster):
+
+    ```bash
+    aws eks describe-cluster --name YOUR-CLUSTER-NAME --query "cluster.resourcesVpcConfig.vpcId" --output text
+    ```
+
+3. Find The CIDR range of VPC using the command below (replacing `VPC-ID` with the ID retrieved in the previous step):
+
+    ```bash
+    aws ec2 describe-vpcs --vpc-ids VPC-ID --query "Vpcs[].CidrBlock" --output text
+    ```
+
+4. Go to the [VPC Console](https://console.aws.amazon.com/vpc/home) and search for the VPC using the ID retrieved in step 2 as shown in the screenshot below:
```

Comment:
We could add VPC Console -> Security Groups to be more clear

Reply:
Agreed, I have specifically mentioned the security group section and changed the URL to go directly there.